papis/commands/tag.py

```diff
@@ -0,0 +1,185 @@
+"""
+This command allows users to add and remove tags to documents in their library.
+
+Examples
+^^^^^^^^
+
+- To add a tag use the `--new` (or `-n`) option:
+
+    .. code:: sh
+
+        papis tag --add TAG1 --add TAG2 QUERY
+
+  This will add TAG1 and TAG2 to a document matched by QUERY. You can repeat
+  `--new` as many times as you need. The query is any query supported by papis.
+  If the query matches more than one document, Papis' picker will be started to
+  let you pick the document from those matched (just as with Papis' other
+  commands).
```

Comment:
```suggestion
- To add a tag use the `--add` (or `-p`) option:

    .. code:: sh

        papis tag --add TAG1 --add TAG2 QUERY

  This will add TAG1 and TAG2 to a document matched by QUERY. You can repeat
  `--add` as many times as you need. The query is any query supported by papis.
  If the query matches more than one document, Papis' picker will be started to
  let you pick the document from those matched (just as with Papis' other
  commands).
```

Reply:
My bad, I could have just searched for those... Thanks!